stew/ranges/stackarrays.nim

```diff
@@ -49,8 +49,11 @@ when defined(windows):
 else:
   proc alloca(n: int): pointer {.importc, header: "<alloca.h>".}
 
+when not declared(RangeDefect):
```

Comment:
it would make sense to create  `stew/shims/stddefects.nim` which creates aliases for all the 1.6-renamed defects

Reply:
Ok, done.